code/game/jobs/job/security.dm

```diff
@@ -215,25 +215,25 @@ var/list/sec_departments = list("engineering", "supply", "medical", "science")
 		switch(department)
 			if("supply")
 				ears = /obj/item/device/radio/headset/headset_sec/alt/department/supply
-				dep_access = list(access_mailsorting, access_mining)
+				dep_access = list(access_mailsorting, access_cargo, access_mineral_storeroom)

```

Comment:
No it wasn't intentional, I'll make key changes to the code for this sometime tomorrow.